Adafruit_NeoPixel.cpp

```diff
@@ -1082,6 +1082,39 @@ uint32_t Adafruit_NeoPixel::Color(uint8_t r, uint8_t g, uint8_t b) {
   return ((uint32_t)r << 16) | ((uint32_t)g <<  8) | b;
 }
 
+// Convert Hue/Saturation/Brightness values to a packed 32-bit RBG color.
+// hue must be a float value between 0 and 360
+// saturation must be a float value between 0 and 1
+// brightness must be a float value between 0 and 1
+uint32_t Adafruit_NeoPixel::HSVColor(float h, float s, float v) {
```

Comment:
Maybe we could use the `constrain` function to ensure the preconditions suggested by this comment?